PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,18 @@
 ## Description
-<!-- Please explain the changes you made here. -->
+
 
 ## Testing done
-<!-- Please explain local testing done, this can include
-automated testing, manual testing, cross browser testing -->
 
-## Testing Plan
-<!-- Please explain testing planned this can include
-automated testing, manual testing, cross browser testing -->
 
 ## Screenshots
-<!-- Please provide screenshots of UI changes if applicable -->
-
-## Acceptance Criteria (Definition of Done)
 
-#### Unique to this PR
-<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
-- [ ] _Replace this line with individual tasks unique to your PR_
 
-#### Applies to all PRs
+## Acceptance criteria
+- [ ]
 
-- [ ] Appropriate logging
+## Definition of done
```

Comment:
From everything I've read, the definition of done is applicable to all PRs and the AC is for individual PRs. Seemed like a good idea to separate them out as such.

Reply:
We can maybe make the distinction clearer with the headings `Acceptance criteria (unique to this PR)` and `Definition of done (applicable to all PRs)`.

Alternatively, we could put text in there that makes sense to include in the PR like
```
## Acceptance criteria
This PR does the following:
- [ ]

## Definition of done
This PR meets the vets-website standards because:
- [ ] A link has been provided to the original...
```